src/test/java/org/eclipse/microprofile/starter/core/TemplateVariableProviderTest.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.microprofile.starter.core;
+
+import org.eclipse.microprofile.starter.core.model.JessieMaven;
+import org.eclipse.microprofile.starter.core.model.JessieModel;
+import org.eclipse.microprofile.starter.core.model.JessieSpecification;
+import org.eclipse.microprofile.starter.core.model.MicroProfileVersion;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.junit.Assert.*;
```

Comment:
CheckStyle fails on star imports.

instead of `assertEquals` you can use `Assert.assertEquals` and use a regular import